lamp.json

```diff
@@ -5,5 +5,6 @@
     "docker_images": [
         "aladdin-demo",
         "aladdin-demo-commands"
-    ]
+    ],
+    "release_tag": "%Y-%m-%d-release"
```

Comment:
This is also a fivestars specific thing. The release plugin is not part of aladdin core. I think maybe we want to have a bigger discussion on what exactly "aladdin demo" is. We could create our own fork with the fivestars specific stuff, although it becomes a slightly bigger pain to manage and update